pyscf/adc/radc.py

```diff
@@ -2308,16 +2307,11 @@ def analyze_eigenvector_ea(adc):
                 orb_d_idx = orb_idx - n_singles
                       
                 i_rem = orb_d_idx % (nvir*nvir)
-                i_idx = (orb_d_idx - i_rem )//(nvir*nvir)
-                temp_doubles_idx[0] = int(i_idx + 1) 
+                i_idx = orb_d_idx//(nvir*nvir)
```

Comment:
Please name all idx and rem variables properly so that they actually correspond to the i, a, b, indices, etc. Please make these changes for RADC and UADC and test everything.